jenkins/development.Dockerfile

```diff
@@ -171,6 +172,7 @@ RUN \
         python3-sphinx \
         `#` \
         `# Install utilities for building RPMs, evaluating test results, etc.` \
+        copr-cli \
```

Comment:
I suppose it will eventually become clear why we need this?

Reply:
Nope, it won't...unless you try to build the Server RPM...at which point you'll need to install this package on the platform on which you're trying to run the build...but, if you're building it in the development container, then, because it will already have been installed by this change, it won't be a problem, and you'll never know!  😀

(The Server's RPM build `Makefile` uses the COPR CLI to initiate a build on COPR, so it relies on having this package installed.  To the extent that you've been building the Server RPMs, you've probably already installed this and long since forgotten.)

Reply:
The real question is whether we really want/need to prepare for building an RPM inside the container.

However, on reflection, it's not a bad idea since "Mr. Jenkins" eventually ought to be doing that for us automatically as part of the test toolchain.